src/applications/edu-benefits/0994/pages/trainingProgramsInformation.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import fullSchema from 'vets-json-schema/dist/22-0994-schema.json';
+import _ from 'lodash';
+import dateUI from 'us-forms-system/lib/js/definitions/date';
+
+const { plannedStartDate, vetTecProgramLocations } = fullSchema.properties;
+
+const trainingDescription = (
+  <div>
+    <p>
+      Now we’ll gather information about the training program(s) you’re
+      interested in attending.
+    </p>
+    <p>You can list up to 3 programs.</p>
+  </div>
+);
+
+export default function VetTecProgramView({ formData }) {
```

Comment:
should this be in a content file?

Reply:
I figured since it wasn't a lot of content...should it be in a separate file? I can put it there, but just wondered.

Reply:
Done